imagebuildah/build.go

```diff
@@ -173,11 +173,9 @@ type BuildOptions struct {
 	OmitTimestamp bool
 	// OS is the specifies the operating system of the image to be built.
 	OS string
-	// MaxPullPushRetries is the maximum number of attempts we'll make to pull or push any one
+	// PullRetryOpts is the options of the maximum number of attempts we'll make to pull or push any one
```

Comment:
Why isn't this jsut RetryOpts for consistency?

Reply:
I would very much like to avoid changing our API here if we can.